x/distribution/client/rest/rest.go

```diff
@@ -14,12 +16,18 @@ import (
 	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
 )
 
+func RegisterHandlers(cliCtx context.CLIContext, m codec.Marshaler, txg tx.Generator, r *mux.Router, queryRoute string) {
```

Comment:
Let's take this as an opportunity to clean this up. None of the handlers need to be passed a `queryRoute`. They should already know the route and that's `types.QuerierRoute`.

Reply:
Sure, that should be good